diff --git a/git_contributions_importer/Importer.py b/git_contributions_importer/Importer.py
index f43c9d3..6271c65 100644
--- a/git_contributions_importer/Importer.py
+++ b/git_contributions_importer/Importer.py
@@ -130,7 +130,7 @@ class Importer:
         for repo in self.repos:
             for b in repo.branches:
                 for c in repo.iter_commits(b.name):
-                    if c.committed_date < ignore_before_date or (self.ignore_before_date != None and c.committed_date < self.ignore_before_date): 
+                    if c.committed_date <= ignore_before_date or (self.ignore_before_date != None and c.committed_date <= self.ignore_before_date): 
                         continue
                     if c.hexsha not in s:
                         s.add(c.hexsha)
